### DISCUSSION TITLE
Combining PKC and Tmdbhelper

### DISCUSSION THREAD

**Discussion by djnield** [croneter/PlexKodiConnect#1382] [Q&A] [answered]
Do we need additional setup or player files to play through tmdbhelper?

If I try play a movie through a helper widget the best I can achieve is some random trailer might play. Usually nothing will happen at all though.

**Comment by croneter** [comment, marked as answer]
What exactly are you trying to do? How do you navigate in Kodi trying to achieve that? (you might want to share some screenshots, too)

**Reply by djnield** [reply]
It's all fine now, apologies.
A reset of the plugin and no more issues now.
